### PR TITLE
Fix: general genric helloworld example

### PR DESCRIPTION
**Pull request by amudong**
Fix:fix general genric helloworld example

### CODE REVIEW

File: examples/general/dubbo/go-server/app/user_provider2.go [hunk 1]
```diff
@@ -21,9 +21,7 @@ import (
 	"context"
 	"fmt"
 	"strconv"
-)
 
-import (
 	"github.com/apache/dubbo-go-hessian2/java_exception"
```
Comment (fangyincheng) [inline examples/general/dubbo/go-server/app/user_provider2.go]: format package. it should be three parts. the first is standard package, the second is third party package, and the third is `github.com/apache/dubbo-go/...`.

File: examples/general/dubbo/go-server/app/user_provider1.go [hunk 2]
```diff
@@ -21,9 +21,7 @@ import (
 	"context"
 	"fmt"
 	"strconv"
-)
 
-import (
```
Comment (fangyincheng) [inline examples/general/dubbo/go-server/app/user_provider1.go]: format package. it should be three parts. the first is standard package, the second is third party package, and the third is `github.com/apache/dubbo-go/...`.

File: examples/helloworld/dubbo/go-server/app/user.go [hunk 3]
```diff
@@ -19,14 +19,10 @@ package main
...
 	"context"
+	"fmt"
 	"time"
-)
-
-import (
-	"github.com/apache/dubbo-go-hessian2"
-)
 
-import (
```
Comment (fangyincheng) [inline examples/helloworld/dubbo/go-server/app/user.go]: don't modify it.

File: examples/generic/go-client/app/client.go [hunk 4]
```diff
@@ -47,8 +51,37 @@ func main() {
...
+		syscall.SIGQUIT, syscall.SIGTERM, syscall.SIGINT)
+	for {
+		sig := <-signals
+		logger.Infof("get signal %s", sig.String())
+		switch sig {
+		case syscall.SIGHUP:
+			// reload()
+		default:
+			go time.AfterFunc(time.Duration(survivalTimeout), func() {
+				logger.Warnf("app exit now by force...")
```
Comment (hxmhlt) [inline examples/generic/go-client/app/client.go]: time.AfterFunc is a new goroutine. So u can remove the 'go'.

**Comment by AlexStocks** [comment]
the target branch should be develop instead of master.

**Pull request closed without merging**